meshtastic/deviceonly.proto

```diff
@@ -24,17 +24,17 @@ message PositionLite {
    * The new preferred location encoding, multiply by 1e-7 to get degrees
    * in floating point
    */
-  sfixed32 latitude_i = 1;
+  optional sfixed32 latitude_i = 1;
 
   /*
    * TODO: REPLACE
    */
-  sfixed32 longitude_i = 2;
+  optional sfixed32 longitude_i = 2;
 
   /*
    * In meters above MSL (but see issue #359)
    */
-  int32 altitude = 3;
+  optional int32 altitude = 3;
```

Comment:
Altitude should certainly be optional (here and other position places), but I'm not completely sure about the other two? I guess it doesn't make much sense if _both_ are 0, ever, but one or the other is obviously possible.